### ISSUE TITLE
The training phase is too long

### ISSUE THREAD

**Issue by ll490187880** [vita-epfl/openpifpaf#117]
The training phase takes several days. Will the distributed training  in multi-machines be added?  Besides, how to decide the epochs based on the numbers of training images. Is it necessary to train 75 epochs?

**Comment by svenkreiss** [comment]
The log files for all training runs for the pre-trained models are public. Also, plots of the logs are in [docs/studies.ipynb](https://github.com/vita-epfl/openpifpaf/blob/master/docs/studies.ipynb):

![image](https://user-images.githubusercontent.com/2775078/63036765-52c2c480-be83-11e9-8f59-ecd639c565c7.png)

You can see the AP as a function of number of epochs and shorten your training. You can also switch to shufflenetv2x2 which is faster to train and much faster at inference time if an AP of about 60% is good enough (I think it usually is).

Training on multiple GPUs in one machine is supported well.
